### ISSUE TITLE
query slashing signing-info: SigningInfo not found

### ISSUE THREAD

**Issue by kenorb** [FunctionX/fx-core#16]
I'm following the command from this page: https://functionx.gitbook.io/home/validators/validator-setup
however I've got the following error:

```
$ fxcored query slashing signing-info "$(fxcored tendermint show-validator)"
Error: rpc error: code = NotFound desc = rpc error: code = NotFound desc = SigningInfo not found for validator fxvalcons1rqetq...: key not found
```

My validator on Testnet shows as active. What I'm missing?

**Comment by zakir-code** [comment]
You can try to check whether the `pub_keys` in these two places are the same.

1. `cat .fxcore/config/priv_validator_key.json | jq '.pub_key'`

2. `fxcored query staking validator <fxvaloper_address> -o json | jq '.consensus_pubkey'`

`fxvaloper_address` can be found in the explorer.

**Comment by zakir-code** [comment]
Check whether the validator is in jail (`jailed: true`) and whether the validator status is `BOND_STATUS_BONDED`.

`fxcored query staking validator <fxvaloper_address>`

**Comment by kenorb** [comment]
I've unjailed validator (which was jailed today), and the problem persist (_SigningInfo not found_).

- `jailed: false`
- `status: BOND_STATUS_BONDED`

The `pub_key` from `priv_validator_key.json` **is different** the one is shown in `query staking validator`. I've checked different backups from 2 weeks time and my `pub_key` didn't change.

They need to be the same? I didn't have problems with unjailing validator, so the authentication works as expected.
